src/app/layout.tsx

```diff
@@ -55,8 +55,7 @@ export default function RootLayout(props: PropsWithChildren) {
       <head>
         <Meta />
         <Script
-          async
-          defer
+          strategy={'afterInteractive'}
```

Comment:
please revert this change, it is already the default strategy
https://nextjs.org/docs/app/api-reference/components/script#strategy

Reply:
You're right!